js/wide-tables.jquery.js

```diff
@@ -83,23 +83,20 @@
         }
       }
 
-      // SBTWO-5105 check tables after load, in case contents cause resize
-      function onLoad() {
-        self.findWideTables(o.container).each(handleTable);
-      }
 
-      $(window).load(onLoad);
+      self.findWideTables(o.container).each(handleTable);
```

Comment:
Don't we lose the fix that the onLoad was added for by doing this, namely that widetables won't be activated if it's caused by large images that haven't yet loaded?